.github/workflows/validate.yml

```diff
@@ -172,7 +172,7 @@ jobs:
 
   validate-old-ghcs:
     name: Validate old ghcs ${{ matrix.extra-ghc }}
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
I wonder what’s up with the comment below that claims the job only runs on 16.04. Should it be removed?

Reply:
I *think* it applies to the docker container, i.e., `phadej/ghc:8.8.4-xenial` is an ubuntu-16.04-based container. https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container

(I'm not entirely clear on which parts run where in that case...)